napi-inl.h

```diff
@@ -196,6 +196,22 @@ struct ThreadSafeFinalize {
   FinalizerDataType* data;
   Finalizer callback;
 };
+
+template <typename ContextType, typename DataType, typename CallJs, CallJs call>
+typename std::enable_if<call != nullptr>::type
+static inline CallJsWrapper(napi_env env, napi_value jsCallback, void *context, void *data) {
+  call(env, Function(env, jsCallback), static_cast<ContextType *>(context),
```

Comment:
Does this need the  if (jsCallback != nullptr) check like the version below or do we expect the CallJs

Reply:
Hi @mhdawson ,

Your comment might have got cut off / incomplete but I think I got you...

I do **not** think we need the `jsCallback != nullptr` check like below. The call will go into the user-defined `CallJs call` with an empty function, which I believe is the normal behavior of using the underlying napi tsfn calls with no callback Function but with a CallJs.